scripts/Update-QDKVersion.ps1

```diff
@@ -41,48 +41,38 @@ $csFiles | ForEach-Object {
     } | Set-Content -Encoding UTF8 $_.Path
 }
 
+function findAndReplace() {
+    param (
+        [string]$stringToReplace,
+        [string]$targetPath
+    )
+
+    (Get-Content -Path $targetPath) | ForEach-Object {
+        $isQuantumPackage = $_ -match $jsonString
```

Comment:
I suspect this should use the parameter rather than the global variable. When you tested it, did this script replace all the occurrences of the old version in all files?
```suggestion
        $isQuantumPackage = $_ -match $stringToReplace
```